docs/NewFeatures/lifecycle_expiration_policies.rst

```diff
@@ -0,0 +1,68 @@
+=============================
+
+Cloud users can apply lifecycle expiration rules (specified in Amazon’s
+`AWS S3 API <https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html>`__)
+to buckets managed through Zenko. These rules are triggered after a defined
+time has passed since the object’s creation. Zenko supports expiration of

```

Comment:
We may mention we can also trigger expiration of current versions based on a date at which the rule starts applying (but no Orbit support as of now, doable with S3 API).

Reply:
"Zenko supports expiration of versioned or non-versioned objects, when a
defined number of days has passed since those objects’ creation. This enables
automatic deletion of older versions of versioned objects to reclaim storage
space. Zenko also supports triggering expiration of current versions on a date
from which a rule applies. This is currently feasible using the S3 API, but not
with Orbit."